karapace/master_coordinator.py

```diff
@@ -175,7 +175,7 @@ def init_schema_coordinator(self):
         self.sc.master_eligibility = self.config["master_eligibility"]
         self.schema_coordinator_ready.set()
 
-    def get_master_info(self) -> Tuple[bool, Optional[str]]:
+    def get_master_info(self) -> Tuple[Optional[bool], Optional[str]]:
```

Comment:
Is the reason why the `bool` is `Optional` because its representing an absence of a value? An `Optional[bool]` can be seen as misleading and it can be argued that just having a bool (if in context there is no real difference between absence of value and value being false) or class with subtypes representing the different states of the value?